### PR TITLE
PKCS11 stack optimization & ClearAllSessions in SetOwner

### PR DESCRIPTION
**Pull request by mykola-kobets-epam**

### CODE REVIEW

**Review by MykolaSolyanko — APPROVED** [review]

File: include/aos/common/config.hpp [hunk 1]
```diff
@@ -240,6 +240,13 @@
 #define AOS_CONFIG_CRYPTO_CSR_PEM_LEN 4096
 #endif
 
+/**
+ * Maximum length of private key in PEM format.
+ */
+#ifndef AOS_CONFIG_CRYPTO_PRIVKEY_PEM_LEN
+#define AOS_CONFIG_CRYPTO_PRIVKEY_PEM_LEN 4096
```
Comment (al1img) [inline include/aos/common/config.hpp:247]: 2048?

File: include/aos/common/pkcs11/pkcs11.hpp [hunk 2]
```diff
@@ -656,6 +656,15 @@ class PrivateKey {
  */
 class Utils {
 public:
+    /**
+     * Maximum size of allocator's memory that is supposed to be reserved for local objects.
+     * Required by some Utils functions like GenerateRSA(ECDSA)KeyPairWithLabel, etc.
+     */
```
Comment (al1img) [inline include/aos/common/pkcs11/pkcs11.hpp:662]: Need mention that additionally required pkcs11::cPrivateKeyMaxSize * num keys for load key API

File: src/common/crypto/mbedtls/cryptoprovider.cpp [hunk 3]
```diff
@@ -245,16 +245,16 @@ Error MbedTLSCryptoProvider::PEMToX509Certs(const String& pemBlob, Array<x509::C
     mbedtls_x509_crt* currentCrt = &crt;
 
     while (currentCrt != nullptr) {
-        x509::Certificate cert;
-
-        auto err = ParseX509Certs(currentCrt, cert);
+        auto err = resultCerts.PushBack({});
```
Comment (al1img) [inline src/common/crypto/mbedtls/cryptoprovider.cpp:248]: EmplaceBack?

File: include/aos/common/config.hpp (hunk 1 above)
Comment (mykola-kobets-epam) [reply include/aos/common/config.hpp]: fixed

File: include/aos/common/pkcs11/pkcs11.hpp (hunk 2 above)
Comment (mykola-kobets-epam) [reply include/aos/common/pkcs11/pkcs11.hpp:664]: fixed

File: src/common/crypto/mbedtls/cryptoprovider.cpp (hunk 3 above)
Comment (mykola-kobets-epam) [reply src/common/crypto/mbedtls/cryptoprovider.cpp]: fixed

**Review by al1img — APPROVED** [review]

**Pull request merged**